Add privacy considerations around private IDs in glue URIs (#15)
mesur-io/draft-zundel-spice-glue-id: draft-zundel-spice-glue-id.md

TODO Security

# Privacy Considerations

## Private identifiers as corporate identifiers

There are some corporate identifers which make use of personal identifiers. This
is the case for registered sole-proprietor businesses in much of the United
States, where the business identifier may be the same as the
social-security-number of the business owner.

It is possible for such identifers to be represented as glue URIs. An
identifier's expression as a glue URI does not change the privacy
characteristics of that identifier. The same cautions and concerns need to be
taken with the glue URI representation as with the original identifier.

> 

Reviewer: ```suggestion
Implementers storing or evaluating glue ids are encouraged to evaluate the privacy characteristics of each authority potentially represented by authority identifiers and to appropriately handle any glue id whose authority identifier format violates privacy policies.
```

TODO Security

# Privacy Considerations

## Private identifiers as corporate identifiers

There are some corporate identifers which make use of personal identifiers. This
is the case for registered sole-proprietor businesses in much of the United
States, where the business identifier may be the same as the
social-security-number of the business owner.

It is possible for such identifers to be represented as glue URIs. An
identifier's expression as a glue URI does not change the privacy
characteristics of that identifier. The same cautions and concerns need to be
taken with the glue URI representation as with the original identifier.

> Implementers storing or evaluating glue ids are encouraged to evaluate the privacy characteristics of each authority potentially represented by authority identifiers and to appropriately handle any glue id whose authority identifier format violates privacy policies.

Author: ```suggestion
Implementers storing or evaluating glue ids are encouraged to evaluate the
privacy characteristics of each identification scheme represented by an authority
identifier and to appropriately handle any glue id which violates privacy policies.
```
